.github/workflows/framework-tests.yaml

```diff
@@ -32,8 +32,46 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Set up Go 1.17
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.17
+
```

Comment:
Is Go needed in this job? Or just in the new `test-real-pebble` one?

Reply:
oops - yeah - I did some refactoring of the workflow and missed cleaning that out.  Fixed.